tests/framework/web/XmlResponseFormatterTest.php

```diff
@@ -57,7 +58,8 @@ public function formatArrayDataProvider()
             [[
                 'a' => 1,
                 'b' => 'abc',
-            ], "<response><a>1</a><b>abc</b></response>\n"],
+                'image:loc' => 'url',
+            ], "<response><a>1</a><b>abc</b><image:loc>url</image:loc></response>\n"],
```

Comment:
That changes format significantly. Likely will result in broken apps.

Reply:
@samdark uses `item` tag instead `image:loc` that's I'm calling "broken app".